manifest.json

```diff
@@ -0,0 +1,246 @@
+{
+  "dependencies": {
+    "thirdParty": {
+      "bleach": {
+        "version": "1.2.2",
+        "zipPath": "https://github.com/jsocol/bleach/archive/v1.2.2.zip",
+        "rootDirPrefix": "bleach-",
+        "targetDirPrefix": "bleach-"
+      },
+      "html5Lib": {
```

Comment:
suggest alphabetizing the keys at the level of bleach, gaeMapReduce, html5Lib, etc. -- makes it easier to find things.